docs/explanation/intro-to-ontologies.md

```diff
@@ -0,0 +1,233 @@
+These identifiers are short ways of referring to IRIs (e.g. CL:000129 = http://purl.obolibrary.org/obo/CL_0000129)
+This IRI is a unique, resolvable identifier on the web.
+A group of ontologies - loosely co-ordinated through the OBO Foundry, have standardised their IRIs (e.g. http://purl.obolibrary.org/obo/CL_0000129  - A term in the cell ontology; http://purl.oblibrary.org/obo/cl.owl - The cell ontology)
+
+#### IRIs? URIs? URLs? 
+- IRI: Internationalised Resource Identifier - a URI that can use characters in multiple languages
+- URI: Unique Resource Identifier - is a string of characters, following a standard specification, that unambiguously identifies a particular (web) resource.
+- URL: Uniform Resource Locator - a web-resolvable URI
+
+## Building scalable ontologies
+
+### Format
+
+OBO ontologies are done mostly in OWL2 or OBO format.
+
+For more indepth explanation of formats (OWL, OBO, RDF etc.) refer to explainer on [OWL format variants](../explanation/owl-format-variants.md)
+
+### An ontology as a classification 
+
+The ontology also functions as a classification.
+Below you will see a classification of parts of the insect and how it is represented in the ontology.
+
+![](../images/discussions/intro-to-ontologies/insect-classification.png)
+
+We use a SubClassOf (or is_a in obo format) to represent entities that are fully encapsulated by the parent class.
+For example: 
+OWL: hindwing SubClassOf wing 
+OBO: hindwing is_a wing
+
+![](../images/discussions/intro-to-ontologies/hindwing-subclass.png)
+
+We use a relation `part_of` to represent an entity that is a part of a whole entity. 
+For example: 
+English: all (insect) legs are part of a thoracic segment
+OWL: 'leg' SubClassOf part_of some thoracic segment
+OBO: 'leg'; relationship: part_of thoracic segment
+Note the existential quantifier `some` in OWL format - this means, that is interpreted as "there exists", "there is at least one", or "for some".
+
+![](../images/discussions/intro-to-ontologies/leg-wing.png)
+
+Note that there is a difference in how you order your subClassOf:
+`'wing' SubClassOf part_of some 'thoracic segment'` is correct
+`'thoracic segment' SubClassOf has_part some 'wing'` is incorrect as it implies all thoracic segment has wings on it. 
+
+Similarly:
+`'claw' SubClassOf connected_to some 'tarsal segment'` is correct
+`'tarsal segment' SubClassOf  connected_to some 'claw'` is incorrect as it implies all tarsal segments are connected to claws (for example some tarsal segments are connected to other tarsal segments)
+
+![](../images/discussions/intro-to-ontologies/tarsal-segments.png)
+
+These relationships store knowledge in a queryable format. For more information about querying, please refer to guide on [DL queries](../tutorial/basic-dl-query.md) and [SPARQL queries](../tutorial/sparql.md)
+
+### Scaling Ontologies
+
+There are many ways to classify stuff - e.g. a neuron can be classified by strucutre, electrophysiology, neurotransmitter, lineage, etc. 
```

Comment:
There are many ways to classify things. For example, a neuron can be classified by structure, electrophysiology, neurotransmitter, lineage, etc.